modules/nf-core/deseq2/differential/tests/main.nf.test

```diff
@@ -157,7 +145,10 @@ nextflow_process {
                     file(rnaseq_test_data_dir + 'SRP254919.samplesheet.csv', checkIfExists: true),
                     file(rnaseq_test_data_dir + 'SRP254919.salmon.merged.gene_counts.top1000cov.tsv', checkIfExists: true)
                 ]
-                ch_spikes = SPOOF_SPIKES.out.map{tuple(['id':'ERCC'], it)}
+                ch_spikes = [
+                    ['id':'ERCC'], 
+                    file(rnaseq_test_data_dir + 'SRP254919.spikes.tsv', checkIfExists: true)
+                ]
```

Comment:
I don't think rnaseq_test_data_dir is the correct path in the modules repo?

Reply:
It's just a convenience thing (read up) - I just didn't want to keep writing `params.modules_testdata_base_path + 'genomics/mus_musculus/rnaseq_expression/`

Reply:
yep, still not a correct variable name for a module. They should be pipeline independent.

Reply:
so `expression_test_data_dir` looks better to me 🙂 (otherwise people might be confused that it is using the test data for the rnaseq pipeline)

Reply:
Oh right, gotcha now, will do.